website/docs/scanner-plugins.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Scanner Plugins
+---
+
+# Motivation
+
+By default, `copa` uses [Trivy](https://github.com/aquasecurity/trivy) to scan container images for vulnerabilities. However, we understand that different organizations have different requirements and may want to use different vulnerability scanners.
+
+Starting with v0.5.0 and later, `copa` offers extensibility to support different vulnerability scanners. Plugin architecture allows users to use the vulnerability scanner of their choice to patch container images without having to modify `copa`'s core codebase.
+
+# Usage
+
+Scanner plugin binaries must be in `$PATH` and should be prefixed with `copa-`. Copa will automatically detect and use the scanner plugin if it is in `$PATH`.
+
+For example, if you have a scanner plugin binary called `copa-foo` in `$PATH`, you can run `copa` with the following command:
+
+```bash
+copa patch --scanner foo --image $IMAGE ...
+```
+
+# Community Scanner Plugins
+
+If you have built a scanner plugin and would like to add it to this list, please submit a PR to update this section with your plugin.
+
+:::tip
+
+If you have any issues with a specific plugin, please open an issue in the applicable plugin's repository.
+
```

Comment:
I'll open a PR to add [anubhav06/copa-grype](https://github.com/anubhav06/copa-grype) to the list, once this PR merges 👍🏻

Reply:
added, thank you!